velox/functions/sparksql/fuzzer/SparkAggregationFuzzerTest.cpp

```diff
@@ -82,6 +82,10 @@ int main(int argc, char** argv) {
           {"skewness", nullptr},
           {"kurtosis", nullptr},
           {"collect_list", makeArrayVerifier()},
+          // Nulls are put first in Spark ascending sorting, while Velox
```

Comment:
After I changed the config, it still fails. I think this is related to the nested null handling behavior difference from DuckDB and Spark. Please see below run for details. As this is still failed, I prefer to update the comment instead of changing the DuckDB config here. Is this way ok to you?
Thanks

![image](https://github.com/user-attachments/assets/436450b8-35d9-45a7-a8ec-6a74c2590ffe)
https://duckdb.org/docs/sql/data_types/struct#comparison-operators
```
I0717 16:11:49.060940 1510097 AggregationFuzzer.cpp:328] ==============================> Started iteration 43 (seed: 2209954389)
I0717 16:11:49.064766 1510097 AggregationFuzzerBase.cpp:422] Executing query plan:
-- Aggregation[1][SINGLE [g0, g1, g2, g3, g4] a0 := min(ROW["c0"])] -> g0:BIGINT, g1:ROW<f0:SMALLINT,f1:SMALLINT>, g2:ARRAY<BIGINT>, g3:TINYINT, g4:BOOLEAN, a0:ROW<f0:REAL,f1:DOUBLE,f2:INTEGER>
  -- Values[0][1000 rows in 10 vectors] -> c0:ROW<f0:REAL,f1:DOUBLE,f2:INTEGER>, g0:BIGINT, g1:ROW<f0:SMALLINT,f1:SMALLINT>, g2:ARRAY<BIGINT>, g3:TINYINT, g4:BOOLEAN
I0717 16:11:49.065847 1512201 Task.cpp:1141] All drivers (1) finished for task test_cursor 764 after running for 1 ms.
I0717 16:11:49.065862 1512201 Task.cpp:1881] Terminating task test_cursor 764 with state Finished after running for 1 ms.
/var/git/velox/velox/exec/tests/utils/QueryAssertions.cpp:1140: Failure
Failed
Expected 994, got 994
2 extra rows, 2 missing rows
2 of extra rows:
        8325638755283651830 | [10783,30894] | null | null | false | [null,0.7618984968867153,2029251925]
        8325638755283651830 | [10783,30894] | [] | null | false | [null,0.4235427004750818,2029251925]

2 of missing rows:
        8325638755283651830 | [10783,30894] | null | null | false | [0.8246623873710632,0.03814789466559887,2029251925]
        8325638755283651830 | [10783,30894] | [] | null | false | [0.5195193290710449,0.39897391432896256,2029251925]

Unexpected results
```

Reply:
@zhli1142015 Thank you for checking. Updating the comment would be fine. Let's also prioritize enabling Spark as source of truth in the Fuzzer to avoid loosing coverage.

Reply:
Updated, thanks.